webui/ocw/lib/db.py

```diff
@@ -19,49 +27,111 @@
 
 
 @transaction.atomic
-def update_or_create_instance(provider, instance_id, region, csp_info, vault_namespace):
+def sync_pcp_to_local_db(pc_instances, provider, vault_namespace):
```

Comment:
```suggestion
def sync_csp_to_local_db(pc_instances, provider, vault_namespace):
```

Reply:
ok , actually I was not sure about name just knew that it should not be the old one :) so any name from is fine for me ... except the old one :D